AspNetCore.CacheOutput/InvalidateCacheOutputAttribute.cs

```diff
@@ -26,17 +26,27 @@ public InvalidateCacheOutputAttribute(string methodName, Type controllerType, Ty
             this.cacheKeyGeneratorType = cacheKeyGeneratorType;
         }
 
-        public override async Task OnActionExecutionAsync(ActionExecutingContext context, ActionExecutionDelegate next)
+        private bool IsFailure(int statusCode)
```

Comment:
1. Reorder public-private methods, private at the bottom.
2. Could you come up with a better name? 300 range, for example, doesn't necessarily mean failure.

Reply:
Done